romana-install/group_vars/aws

```diff
@@ -7,5 +7,18 @@ instance_type:
     devstack: "t2.small"
     kubernetes: "t2.small"
 
-controller_image: "ami-bf3ec1fb" # Ubuntu 14.04.3 LTS
-compute_image: "ami-bf3ec1fb" # Ubuntu 14.04.3 LTS
+distro_info:
+  "ubuntu":
+    ami:  "ami-bf3ec1fb" # Ubuntu 14.04.3 LTS
+    user: "ubuntu"
+    bootstrap:
```

Comment:
The use of bootstrap here to list commands that are later expanded into a shell or template is certainly unusual. I'm not saying this is good or bad, just unusual.